Modules/EMCAL/src/RawTask.cxx

```diff
@@ -378,8 +378,14 @@ void RawTask::monitorData(o2::framework::ProcessingContext& ctx)
     nchannels[i] = 0;
 
   // Accept only descriptor RAWDATA, discard FLP/SUBTIMEFRAME
-  std::vector<framework::InputSpec> filter{ { "filter", framework::ConcreteDataTypeMatcher(dataOrigin, "RAWDATA") } };
-  for (const auto& rawData : framework::InputRecordWalker(ctx.inputs(), filter)) {
+  std::cout << "Found " << ctx.inputs().size() << " inputs" << std::endl;
+
+  auto posReadout = ctx.inputs().getPos("readout");
+  auto nslots = ctx.inputs().getNofParts(posReadout);
+  for (decltype(nslots) islot = 0; islot < nslots; islot++) {
+    auto rawData = ctx.inputs().getByPos(posReadout, islot);
+    auto datahead = framework::DataRefUtils::getHeader<header::DataHeader*>(rawData);
+    std::cout << "Next input " << rawData.spec->binding << "" << datahead->dataOrigin.str << "/" << datahead->dataDescription.str << ", subspec " << datahead->subSpecification << std::endl;
```

Comment:
That might not go to infologger, if you need it.

Reply:
This is a leftover debug message - should be removed.